graphql/resolvers.go

```diff
@@ -3101,6 +3101,21 @@ func (r *versionResolver) ChildVersions(ctx context.Context, v *restModel.APIVer
 		for _, cp := range childPatchIds {
 			v, err := r.Query().Version(ctx, cp)
```

Comment:
nit. Can we add a comment indicating this calls the gql resolver and not a db query.